CHANGELOG.rst

```diff
@@ -0,0 +1,9 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package ament_nodl
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.0 (2020-05-28)
+------------------
+* update doc to allow multiple nodl xml files
+* initial commit
```

Comment:
We can probably just start with "initial release" here, honestly.

Reply:
Fixed